vcd/resource_vcd_vapp_org_network_test.go

```diff
@@ -0,0 +1,202 @@
+// +build network vapp ALL functional
+
+package vcd
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccVcdVappOrgNetwork_NotFenced(t *testing.T) {
+	if vcdShortTest {
+		t.Skip(acceptanceTestsSkipped)
+		return
+	}
+
+	resourceName = "TestAccVcdVappOrgNetwork_NotFenced"
+
+	var params = StringMap{
+		"Org":                         testConfig.VCD.Org,
+		"Vdc":                         testConfig.VCD.Vdc,
+		"resourceName":                resourceName,
+		"vappName":                    vappNameForNetworkTest,
+		"EdgeGateway":                 testConfig.Networking.EdgeGateway,
+		"NetworkName":                 "TestAccVcdVAppNetNotFenced",
+		"orgNetwork":                  "TestAccVcdVAppNetNotFenced",
+		"firewallEnabled":             "false",
+		"firewallEnabledForUpdate":    "false",
+		"natEnabled":                  "false",
+		"natEnabledForUpdate":         "false",
+		"retainIpMacEnabled":          "false",
+		"retainIpMacEnabledForUpdate": "true",
+		"isFenced":                    "false",
+		"isFencedForUpdate":           "true",
+		"FuncName":                    "TestAccVcdVappOrgNetwork_NotFenced",
+	}
+
+	rungVappOrgNetworkTest(t, params)
+}
+
+func TestAccVcdVappOrgNetwork_Fenced(t *testing.T) {
+	if vcdShortTest {
```

Comment:
Why are we skipping here? This will prevent the test from running in the binary tests

Reply:
updated